versioned_docs/version-2.0/self_hosting/configuration/blob_storage.mdx

```diff
@@ -111,3 +112,74 @@ If using an access key and secret, you can also provide an existing Kubernetes s
 This is recommended over providing the access key and secret key directly in your config.
 
 :::
+
+## TTL Configuration
+
+If using the [TTL](/self_hosting/configuration/ttl) feature with LangSmith, you'll also have to configure TTL rules for
+your blob storage. Trace information stored on blob storage is stored on a particular prefix path, which determines the TTL for the
+data. As traces are upgrades, they are place in the upgrade prefix path as well to ensure that the data is upgraded in
+blob storage as well.
+
+The following TTL prefix are used:
+
+- `ttl_s/`: Short term TTL, configured for 15 days.
+- `ttl_l/`: Long term TTL, configured for 401 days.
```

Comment:
Worth noting 1 day longer than TTLs in app?

Reply:
and if that you customize them, then you should also follow the same pattern?

Reply:
^ agreed with adding note on customizing

Reply:
We actually cutoff the TTL display from clickhouse via query, so I will actually set it to be same as clickhouse.  Added note on customization.